docs/pages/code/operations/averages.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App;
+
+use loophp\collection\Collection;
+
+include __DIR__ . '/../../../../vendor/autoload.php';
+
+$collection = Collection::fromIterable([1, 2, 3, 4, 5])
+    ->averages(); // [1, 1.5, 2, 2.5, 3]
```

Comment:
here we could give an example of the calculations, like:

```php
// [1, 1.5, 2, 2.5, 3]
// [1/1, (1+2)/2, (1+2+3)/3 ...]
```

Reply:
Done.